test/pummel/test-policy-integrity.js

```diff
@@ -109,7 +109,7 @@ function drainQueue() {
       `deletable-policy-${testId}.json`
     );
     const cliPolicy = willDeletePolicy ? tmpPolicyPath : policyPath;
-    fs.rmdirSync(configDirPath, { maxRetries: 3, recursive: true });
+    fs.rmSync(configDirPath, { maxRetries: 3, recursive: true, force: true });
```

Comment:
we might want to leave a test for `rmmdirSync` too for the time being, since we've indicated that it will be supported for the duration of Node 15.

Reply:
I removed usages of `rmdir` because it now warns in some cases. It does still have its own test suite.